django_project/frontend/src/pages/Dashboard/LeftPanel/ContextLayers/index.jsx

```diff
@@ -94,14 +94,14 @@ function ContextLayers() {
   }, [layers, selectedLayer])
 
   const initialize = (_contextLayers) => {
-    if (selectedLayer.length > 0) {
-      selectedLayer.forEach(layer => {
-        dispatch(
-          Actions.Map.removeContextLayer(layer)
-        )
-      })
-    }
-    setSelectedLayer(contextLayers.filter(row => row.visible_by_default).map(row => row.id + ''))
+    // if (selectedLayer.length > 0) {
+    //   selectedLayer.forEach(layer => {
+    //     dispatch(
+    //       Actions.Map.removeContextLayer(layer)
+    //     )
+    //   })
+    // }
+    // setSelectedLayer(contextLayers.filter(row => row.visible_by_default).map(row => row.id + ''))
```

Comment:
We need to disable this code, but I'm not sure about the consecuences.

Reply:
https://github.com/unicef-drp/GeoSight-OS/blob/fa1304ecdf0f8b7ff60fae692de23024ceffc881/django_project/frontend/src/pages/Dashboard/LeftPanel/ContextLayers/index.jsx#L72

We need this to delete all current context layer that rendered. 
When on config a project, we can delete or add new one, and we need to delete the context layers that are deleted

Why we need to delete this code?

Reply:
I just checked again with the redux dev tools when check and uncheck a context layer. The actions `ADD_CONTEXTLAYERS` and `REMOVE_CONTEXTLAYERS` was fired without problem with this code commented. I no other reason, I will delete after this comprobation.